packages/app-check/src/internal-api.test.ts

```diff
@@ -197,35 +196,48 @@ describe('internal api', () => {
       expect(errorFn1.args[0][0].name).to.include('exchange error');
     });
 
+    it('ignores listeners that throw', async () => {
+      activate(app, FAKE_SITE_KEY, false);
+      stub(reCAPTCHA, 'getToken').resolves(fakeRecaptchaToken);
+      stub(client, 'exchangeToken').resolves(fakeRecaptchaAppCheckToken);
+      const listener1 = (): void => {
+        throw new Error();
+      };
+      const listener2 = spy();
+
+      const errorFn1 = spy();
+
+      addTokenListener(app, fakePlatformLoggingProvider, listener1, errorFn1);
+      addTokenListener(app, fakePlatformLoggingProvider, listener2);
+
+      await getToken(app, fakePlatformLoggingProvider);
+
+      expect(errorFn1).not.to.be.called;
```

Comment:
Probably worth checking that listener1 is called.

Reply:
Added.